### PR TITLE
[IMP] Module Estate/ Estate Account: Exercises of Chapters 12 -14

### PR DESCRIPTION
**Pull request by Emilioglez873**
# Development of chapter 12, 13 and 14 of tutorial server framework 101 in [Odoo docs ](https://www.odoo.com/documentation/17.0/developer/tutorials/server_framework_101.html)

Model inheritance in res.users to show the properties sell by sales person
Adding method with decorator @ondelete, to prevent deleting properties which are not new or canceled
View inheritance in base.view_users_form to show the field property_ids
Creating link module between estate and account modules
Overriding the action_sold method of estate.property to create an invoice
Adding two invoice lines for each property sold:
- 6% of the selling price
- an additional 100.00 from administrative fees

Adding kanban view for estate.property with grouping by property type

### CODE REVIEW

**Comment by sebasdrk17** [comment]
@Emilioglez873 rebase please

**Comment by Emilioglez873** [comment]
@sebasdrk17  Listo ya hice el rebase y también jale los cambios de master

File: estate/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-    'data': [
-        "security/ir.model.access.csv",
+    "data": [
+        "views/res_users_views.xml",
         "views/estate_property_views.xml",
         "views/estate_property_offer.xml",
         "views/estate_type_views.xml",
         "views/estate_menus.xml",
+        "security/ir.model.access.csv",
+
```
Comment (sebasdrk17) [inline estate/__manifest__.py]: remove extra space

**Review by sebasdrk17 — CHANGES_REQUESTED** [review]
Code looks ok, just run pre-commit to check the linter

There are some functional errors that I'll leave in a new comment

**Comment by sebasdrk17** [comment]
# Functional issues:

1. There is an error when trying to install the module I've already solve it but I want you to get the error, to do this drop the database and create a new DB
2. The header options have test names, they should have proper names

Screenshot 2024-10-08 at 2 59 31 p m


### Property types

Why does it look like this?

Screenshot 2024-10-08 at 3 00 43 p m

Screenshot 2024-10-08 at 3 01 36 p m

**Review by sebasdrk17 — APPROVED** [review]

**Comment by sebasdrk17** [comment]
LGTM

**Comment by sebasdrk17** [comment]
Remove the draft text from the merge request and use [IMP], we are not adding any modules here

**Comment by sebasdrk17** [comment]
@Emilioglez873 

Remember the structure of the commit message: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#commit-message

In short it should be:

[IMP] estate, estate_account: Exercises of Chapters 12 -14

**Pull request merged**